### ISSUE TITLE
[Bug] Collection variables are not being applied to GraphQL request variables

### ISSUE THREAD

**Issue by joobus** [usebruno/bruno#789]
When making a graphql request, if the `Variables` section contains a reference to collection variables, e.g.:
```
{
  "latitude": {{latitude}},
  "longitude": {{longitude}},
  "radius": 30
}
```

The request will fail with the error message `Error invoking remote method 'send-http-request': SyntaxError: Expected property name or '}' in JSON at position 17`.

If the variables are replaced with numerical values, the request succeeds.

**Comment by adarshlilha** [comment]
@helloanoop can I take this up? I am yet to peek into the stack trace
any pointers would be helpful, thanks!

**Comment by sanjai0py** [comment]
Closing this issue as the [PR](https://github.com/usebruno/bruno/pull/887) fixing the issue had already been merged and was released in [v1.3.0](https://github.com/usebruno/bruno/releases/tag/v1.3.0).
